specification/cognitiveservices/data-plane/TextTranslation/Readme.md

```diff
@@ -0,0 +1,3 @@
+# Microsoft Translator API V3
```

Comment:
Readme.md => readme.md :)
This file is not a simply readme file. This is actually an configuration/metadata file for autorest.

Please follow the examples in
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/Face/readme.md 
to write an autorest compatible readme.

Reply:
Yes please be sure to use lower-case in the file name.  Also the file format is incorrect and is still causing breaks, please see the logs for details.